public/app/assetlibrary/item/item.html

```diff
@@ -21,24 +21,33 @@
 
     <!-- EDIT DETAILS -->
     <div class="col-xs-{{ (2 * upperRightButtonCount) + 1 }} col-sm-{{ (2 * upperRightButtonCount) + 1 }} text-right">
-      <a ui-sref="assetlibrarylist.item.edit({assetId: asset.id})" class="btn btn-default" data-ng-if="canManageAsset()">
+      <a ui-sref="assetlibrarylist.item.edit({assetId: asset.id})" class="btn btn-default"
+        data-ng-if="canManageAsset()">
         <i class="fa fa-pencil"></i>
         <span>Edit details</span>
       </a>
-      <a class="assetlibrary-item-pin btn btn-default" data-ng-click="pin()" class="btn btn-default">
+      <button id="pin-asset" class="assetlibrary-item-pin btn btn-default"
+        data-ng-click="pin()">
```

Comment:
**Note:** I took this opportunity to make 'pin' a button and a static `id` for all.